marlowe-runtime/src/Language/Marlowe/Runtime/Transaction/Server.hs

```diff
@@ -217,12 +226,12 @@ execCreate solveConstraints loadWalletContext networkId era mStakeCredential ver
         ScriptDataInBabbageEra -> \(TxOut address _ _ _) -> address == AddressInEra (ShelleyAddressInEra ShelleyBasedEraBabbage) scriptAddress
 
 execApplyInputs
-  :: SlotConfig
```

Comment:
I see now that `IsMarloweVersion` is not really useful downstream. @jhbertra Should I drop this experiment or I can leave it as it is?

Reply:
Up to you - I've found that 90% of the time, passing a `MarloweVersion v` is more helpful than requiring a `IsMarloweVersion v` constraint. The big exception is in class instance headers, where `IsMarloweVersion v` can be very useful.